src/main/scala/com/gu/Config.scala

```diff
@@ -88,15 +88,16 @@ object Config {
         case _ => "2c92c0f965f2121e01660fb1f1057b1a"
       }
 
-    // Do not remove Holiday and Retention Discounts
+    // Do not remove Holiday and Retention Discounts (Cancellation Save Discount)
```

Comment:
@paulbrown1982 - can you confirm that these are the only discounts that we should be leaving in place?